src/waveform/renderers/allshader/waveformrendermark.h

```diff
@@ -95,14 +95,14 @@ class allshader::WaveformRenderMark : public ::WaveformRenderMarkBase,
 
     DigitsRenderNode* m_pDigitsRenderNode{};
 
-    QColor m_playMarkerForegroundColor;
-    QColor m_playMarkerBackgroundColor;
-
-    bool m_untilMarkShowBeats;
-    bool m_untilMarkShowTime;
-    Qt::Alignment m_untilMarkAlign;
-    int m_untilMarkTextSize;
-    float m_untilMarkTextHeightLimit;
+    QColor m_playMarkerForegroundColor{};
+    QColor m_playMarkerBackgroundColor{};
```

Comment:
The initializer is redundant. 
```suggestion
    QColor m_playMarkerForegroundColor;
    QColor m_playMarkerBackgroundColor;
```